deployment/terraform/create.go

```diff
@@ -127,6 +147,25 @@ func (t *Terraform) Create() error {
 	// Updating the nginx config on proxy server
 	t.setupProxyServer(output, extAgent)
 
+	time.Sleep(30 * time.Second)
```

Comment:
Why is this needed ? Did you face any issues where mattermost wasn't yet up when you were trying to create the sysadmin ?

Reply:
Yes, I was getting `109` error return or smth like that. After a brief diving into mm store code, it means `cannot create index error`. But I don't remember if I tested after this patch 🤔 Will test it 👍

Reply:
That's okay for now. But let's have a check that pings the `curl http://localhost:8065/api/v4/system/ping` endpoint and checks for a proper response before proceeding. Created https://mattermost.atlassian.net/browse/MM-23566 for this.